.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: objective-c
+osx_image: xcode9
+rvm:
+  - 2.2.2
+
+env:
+  global:
+    - WORKSPACE="LonaStudio.xcworkspace"
+    - OSX_APP_SCHEME="LonaStudio"
+    - OSX_SDK=macosx10.13
+  matrix:
+    - DESTINATION="arch=x86_64"                      SCHEME="$OSX_APP_SCHEME"           SDK="$OSX_SDK"     RUN_TESTS="YES"    BUILD_EXAMPLE="YES"
```

Comment:
rather than running tests and building the example in the same job, this should be split into multiple jobs - either by using the matrix (if it's important to build the example on every ruby version) or by using `include:` (ie, explicitly included builds)

Reply:
You're right. I've just updated the matrix in order to support parallel jobs 👍